diagnostic.md

```diff
@@ -32,23 +44,35 @@ Give two examples of jQuery getter methods and two examples of jQuery setter
 methods.
 
 <!-- your answer starts here -->
-
+(css) and (attr) are both getters and setters
+clarity 4
+comfort 2
+googled
 <!-- your answer ends here -->
 
 ## Question 5
 
 In your own words, please answer the following question: WHAT IS THE DOM?
 
 <!-- your answer starts here -->
+The DOM is the document object model and a platform and language neutral interface that allows
```

Comment:
DOM is the visual representation of the code which a server sends and what the browser renders